index.html

```diff
@@ -4,9 +4,37 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles/style.css">
     <title>Document</title>
+
 </head>
 <body>
-    New text 
+    <div class="container">
+        
+            <h1>What do we offer</h1>
+
+            <hr>
+            
+        <div class="image-item">
+
+            <div class="support"><a href="#"><img src="/img/Vector (4).svg" alt="support"></a>
```

Comment:
The img is not loaded. pls, use `img/Vector (4).svg` or `./img/Vector (4).svg` paths.
also use alts to retell what is shown in the picture. in other pictures the same alt but other images